src/components/content-tab-assistant.tsx

```diff
@@ -173,35 +214,39 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 	const { __ } = useI18n();
 	const lastMessage = messages.length === 0 ? undefined : messages[ messages.length - 1 ];
 
+	const [ errorMessageId, setErrorMessageId ] = useState< number | null >( null );
```

Comment:
As shared above, we could consider simply keeping a boolean in the component state. We could have something like:
```suggestion
	const [ hasError, setHasError ] = useState( false );
```